### PR TITLE
Add the ability to output meso system logs

### PR DESCRIPTION
**Pull request by dwhenry**
These can be useful for debugging

### CODE REVIEW

File: logs [hunk 1]
```diff
@@ -24,41 +25,74 @@ class Logs
...
-      exit 1
+    if system_log
+      output_system_log
+    else
+      output_app_log
     end
+  end
+
+  def output_system_log
+    hosts = get_hosts('marathon-consul/marathon-consul')
```
Comment (Mongey) [inline logs]: Why does it need to look for this applications host?

File: logs [hunk 2]
```diff
@@ -80,6 +115,7 @@ class Logs
    -h, --help       Show this help.
    -t, --tail       Tail to log out as a continuous stream.
    -a, --apps       List available apps
+   -s, --system     Sysytem log
```
Comment (Mongey) [inline logs]: 💅 "System log"

File: logs (hunk 1 above)
Comment (dwhenry) [reply logs]: It needs an app to lookup the hostname - the apps endpoint doesn't include this info.
The hostname is needed in order to make future requests. 
I have used the marathon-consul as I believe it is one of the first things to start and so was a good choice, happy to do this differently if you can think of a better approach. :)

File: logs (hunk 2 above)
Comment (dwhenry) [reply logs]: Corrected

**Comment by dwhenry** [comment]
@Mongey any thoughts on this PR

File: logs (hunk 1 above)
Comment (Mongey) [reply logs]: Can we use `mesos-proxy/web` instead, `marathon-consul` is not an application that others may be using.

**Comment by Mongey** [comment]
LGTM, if we can swap the application name to be the server of this repo

**Pull request merged**